### PR TITLE
Clio 41610 colour updates

### PR DESCRIPTION
**Pull request by frankieyan**

### CODE REVIEW

File: themis_components/thAutocomplete/index.scss [hunk 1]
```diff
@@ -155,7 +155,7 @@
         height: 100%;
         margin-left: 2px;
 
-        border-right: 1px solid #428bca;
+        border-right: 1px solid $border-color;
```
Comment (PANCAKES) [inline themis_components/thAutocomplete/index.scss]: #428bca and #d1d1d1 both map to ```$border-color``` now? one is blue and the other is light grey

File: themis_components/thTable/index.scss [hunk 2]
```diff
@@ -17,14 +17,14 @@ th-table {
...
-    background: #fff;
+    background: $white;
     border-collapse: collapse;
     width: 100%;
     font-size: inherit;
     margin-bottom: $th-table-margin-bottom;
 
     tbody {
-      border-bottom: 1px solid #d9dada;
+      border-bottom: 1px solid $grey-4;
```
Comment (PANCAKES) [inline themis_components/thTable/index.scss]: should this be ```$panel-border-color```?

File: themis_components/thTable/index.scss [hunk 3]
```diff
@@ -65,13 +65,13 @@ th-table {
       }
 
       &.th-table-sortable.th-table-sort-ascending .th-table-sort-icon {
-        color: #b8b8b6;
+        color: $grey-6;
```
Comment (PANCAKES) [inline themis_components/thTable/index.scss]: mapping doc says ```$grey-3```

File: themis_components/thTable/index.scss [hunk 4]
```diff
@@ -65,13 +65,13 @@ th-table {
...
         visibility: visible;
       }
 
       &.th-table-sortable.th-table-sort-descending .th-table-sort-icon {
         @extend .fa-caret-up;
-        color: #b8b8b6;
+        color: $grey-6;
```
Comment (PANCAKES) [inline themis_components/thTable/index.scss]: doc says ```$grey-3```

File: themis_components/thTable/index.scss [hunk 5]
```diff
@@ -81,7 +81,7 @@ th-table {
     }
 
     tr {
-      border-top: 1px solid #d9dada;
+      border-top: 1px solid $grey-4;
```
Comment (PANCAKES) [inline themis_components/thTable/index.scss]: map doc says ```$panel-border-color```

File: themis_components/thTable/index.scss [hunk 6]
```diff
@@ -90,22 +90,22 @@ th-table {
...
       border: 0 none;
-      color: $light-grey;
+      color: $grey-5;
       margin: 0;
       vertical-align: top;
     }
 
     .th-table-hover-row {
-      background: #fafdff;
+      background: $blue-light;
```
Comment (PANCAKES) [inline themis_components/thTable/index.scss]: does ```$blue-light``` still exist? or is it deprecated?

File: themis_components/thAutocomplete/index.scss (hunk 1 above)
Comment (frankieyan) [inline themis_components/thAutocomplete/index.scss]: I was getting super confused here - but these look like carets that go before or after an item in a multiple autocomplete. I don't think they're being used.

File: themis_components/thTable/index.scss (hunk 2 above)
Comment (frankieyan) [inline themis_components/thTable/index.scss]: Might make sense to create a `$table-border-color` for this.

File: themis_components/thTable/index.scss (hunk 5 above)
Comment (frankieyan) [inline themis_components/thTable/index.scss]: Created $table-border-color as well now

File: themis_components/thTable/index.scss (hunk 3 above)
Comment (frankieyan) [inline themis_components/thTable/index.scss]: `$grey-4` now

File: themis_components/thAutocomplete/index.scss (hunk 1 above)
Comment (frankieyan) [inline themis_components/thAutocomplete/index.scss]: OHH these carets are for drag and drop. Using `$blue` here instead. cc @gthierman

File: themis_components/thComponentExample/index.scss [hunk 7]
```diff
@@ -27,20 +27,20 @@ th-component-example,
...
+      border-right: solid 1px $deprecated-light-blue;
 
       cursor: pointer
     }
 
     input[type="submit"] {
       float: right;
       border: none;
-      border-left: solid 1px $light-blue;
+      border-left: solid 1px $deprecated-light-blue;
```
Comment (mbuckley) [inline themis_components/thComponentExample/index.scss]: @frankieyan What is the plan for these deprecated colors? Can we not remove or update these to actual apollo pallets vars?

File: themis_components/thTextarea/index.scss [hunk 8]
```diff
@@ -24,28 +24,28 @@ th-textarea {
...
       box-shadow: 0 0 10px rgba($blue-shadow, .7);
-      border-color: $blue-tint;
+      border-color: $blue;
     }
 
     &.disabled {
-      background: $grey-90;
-      color: $grey-40;
+      background: $input-disabled-background-color;
+      color: $grey-5;
```
Comment (mbuckley) [inline themis_components/thTextarea/index.scss]: I'm getting a bit tripped up on the use of the semantic vars. For example, in &.disabled we are mixing the use, which may be fine. Just wanted to call it out and see if we wanted to document when to use a semantic var vs a straight colour var.

File: themis_components/thComponentExample/index.scss (hunk 7 above)
Comment (frankieyan) [inline themis_components/thComponentExample/index.scss]: This one would be replaced by the new design Geoff is working on - so rather than spending the time to remap some of these now, we'll just stop further usages with the 'decrecated' prefix and revisit this when reimplementing the docs design.

File: themis_components/thTextarea/index.scss (hunk 8 above)
Comment (JefCurtis) [inline themis_components/thTextarea/index.scss]: I agree with @mbuckley here. Mentioned it above as well. Kind of confusing.

File: themis_components/thWithLabel/index.scss [hunk 9]
```diff
@@ -51,14 +51,14 @@ label.th-label {
       font-size: $primary-text-font-size;
       font-weight: normal;
       vertical-align: middle;
-      color: $primary-text-color;
+      color: $text-color;
```
Comment (JefCurtis) [inline themis_components/thWithLabel/index.scss]: There is only one text color now?

File: themis_components/thWithMessages/index.scss [hunk 10]
```diff
@@ -7,10 +7,10 @@
...
     margin-bottom: 10px;
 
-    .message-container {
-      color: $red-alert;
-      font-size: 13px;
-      font-weight: 400;
-    }
+  .message-container {
+    color: $alert-color;
+    font-size: 13px;
```
Comment (JefCurtis) [inline themis_components/thWithMessages/index.scss]: Should font sizes not have a variable value?

File: themis_theme/colors.scss [hunk 11]
```diff
@@ -1,325 +1,59 @@
...
-/*--------------------------------------------------
-* TYPOGRAPHY
-*--------------------------------------------------*/
-$base-font-size: 10px; //used in the rem mixin and for declaring the root font-size on the html element.
-$primary-text-font-size: 13px;
-$subtext-font-size: 11px;
-$sans-serif: "Helvetica Neue",Helvetica,Arial,sans-serif;
-$gibson-semibold: 'GibsonSemiBold',Geneva,Tahoma,Verdana,sans-serif;
-$gibson-regular: 'GibsonRegular',Geneva,Tahoma,Verdana,sans-serif;
+$grey-6: #263238;
```
Comment (JefCurtis) [inline themis_theme/colors.scss]: Maybe a comment explaining light --> dark or dark --> light?

File: themis_theme/colors.scss [hunk 12]
```diff
@@ -1,325 +1,59 @@
...
-  @extend .antialiased;
-}
-
-//--------------------------------------------------
-// COLOURS
-//--------------------------------------------------
 $yellow: #f6d654;
-$yellow-tint: tint($yellow, 10%);
-$yellow-shade: shade($yellow, 10%);
+$yellow-shade: #ddc14c;
```
Comment (JefCurtis) [inline themis_theme/colors.scss]: why the change in naming here? Should it be `$yellow-dark` or something?

File: themis_theme/colors.scss [hunk 13]
```diff
@@ -1,325 +1,59 @@
...
 
-$blue: #4891dc;
-$blue-tint: tint($blue, 10%);
-$blue-shade: shade($blue, 10%);
-$blue-shadow: rgba(97, 179, 227, .7);
+$orange: #FF8800;
```
Comment (JefCurtis) [inline themis_theme/colors.scss]: Dafuuuuuuck??

File: themis_theme/typography.scss [hunk 14]
```diff
@@ -186,3 +189,20 @@
   -moz-osx-font-smoothing: grayscale;
   text-shadow: 1px 1px 1px rgba(0,0,0,0.004);
 }
+
+::-moz-placeholder {
+  opacity: 1;
```
Comment (JefCurtis) [inline themis_theme/typography.scss]: You mentioned that you might move the `opacity` change to all of them?

File: themis_theme/variables.scss [hunk 15]
```diff
@@ -0,0 +1,226 @@
...
+$min-width: 1060px;
+$firm-name-width-large: 1300px;
+$firm-name-width-medium: 800px;
+$firm-name-width-small: 340px;
+$firm-name-width-xsmall: 300px;
+
+/*--------------------------------------------------
+* CSS3 Mixins
+*--------------------------------------------------*/
+@mixin box-sizing($border-box: border-box) {
```
Comment (JefCurtis) [inline themis_theme/variables.scss]: I'm a little confused about this whole file here. Is it needed?

File: themis_theme/colors.scss (hunk 13 above)
Comment (frankieyan) [inline themis_theme/colors.scss]:
```scss
#noauth .password-strength-indicator.weak {
  color: $orange;
  border-color: $orange;
}
```
😉

File: themis_theme/colors.scss (hunk 12 above)
Comment (frankieyan) [inline themis_theme/colors.scss]: 👍 Let's call this yellow-dark

File: themis_components/thTextarea/index.scss (hunk 8 above)
Comment (frankieyan) [inline themis_components/thTextarea/index.scss]: I know - but I'm desperately trying to regain control of the scope of this ticket here

File: themis_theme/typography.scss (hunk 14 above)
Comment (frankieyan) [inline themis_theme/typography.scss]: When we take out the vendor prefixes yes. But I'd prefer to wait til autoprefixer is being run on the ThemisUI docs and when it does reliably add the vendor prefix for Edge.

File: public/stylesheets/docs-app.scss [hunk 16]
```diff
@@ -79,17 +79,17 @@ h1, h2, h3 {
...
     line-height: $header-height;
     text-align: center;
     text-decoration: none;
 
-    border-top: solid 1px $grey-70;
-    border-right: solid 1px $grey-70;
-    background-color: $grey-90;
+    border-top: solid 1px $grey-4;
+    border-right: solid 1px $grey-4;
+    background-color: $grey-2;
```
Comment (frankieyan) [inline public/stylesheets/docs-app.scss]: I think @gthierman has them memorized by now at this point haha.

File: themis_theme/colors.scss (hunk 13 above)
Comment (JefCurtis) [inline themis_theme/colors.scss]: Weird.

File: themis_components/thAutocomplete/index.scss [hunk 17]
```diff
@@ -143,7 +143,7 @@
         height: 100%;
         margin-right: 2px;
 
-        border-left: 1px solid #428bca;
+        border-left: 1px solid $blue;
```
Comment (frankieyan) [inline themis_components/thAutocomplete/index.scss]: Hm its a drag-and-drop target cursor for ui-select - not sure if this deserves its own variable at this point.

File: themis_theme/variables.scss (hunk 15 above)
Comment (JefCurtis) [inline themis_theme/variables.scss]: If we need to clean this up later we should probably create a ticket with a timer on it so we can come back to this a review. What do you think?

File: themis_components/thTextarea/index.scss (hunk 8 above)
Comment (frankieyan) [inline themis_components/thTextarea/index.scss]: The plan is to create a confluence page in the meantime while the colour docs pages on ThemisUI is being built. This will be sent out to the rest of product and it will outline the steps needed to submit a PR here to get new semantic variables created.

File: public/stylesheets/docs-app.scss [hunk 18]
```diff
@@ -79,17 +79,17 @@ h1, h2, h3 {
...
-    color: $grey;
+    color: $grey-6;
     line-height: $header-height;
     text-align: center;
     text-decoration: none;
 
-    border-top: solid 1px $grey-70;
-    border-right: solid 1px $grey-70;
-    background-color: $grey-90;
+    border-top: solid 1px $grey-4;
```
Comment (frankieyan) [inline public/stylesheets/docs-app.scss]: Replaced all `1px solid $grey-4` with `$border`, and `1px solid $grey-5` on `thRadio` to `$border-radio`

File: themis_components/thButton/index.scss [hunk 19]
```diff
@@ -8,7 +8,7 @@
       background-color: $hover-color;
 
       &:active {
-        background-color: mix($hover-color, #000, 85%);
+        background-color: mix($hover-color, $black, 85%);
```
Comment (frankieyan) [inline themis_components/thButton/index.scss]: Consulted Geoff on this one - an explicit colour will be defined when this gets converted to Apollo but for our current purpose (darkening the active state) it's fine.

File: themis_components/thTextarea/index.scss (hunk 8 above)
Comment (frankieyan) [inline themis_components/thTextarea/index.scss]: Created `$input-disabled-text-color` here for `$grey-5`

File: themis_theme/variables.scss (hunk 15 above)
Comment (frankieyan) [inline themis_theme/variables.scss]: Ticket created in the font epic

File: themis_components/thContextualMessage/index.scss [hunk 20]
```diff
@@ -7,10 +7,10 @@
   margin-top: 1.0rem;
   padding: 1rem;
 
-  color: $grey-95;
+  color: $grey-2;
 
   border-radius: $border-radius;
-  background-color: rgba($grey, 0.9);
+  background-color: rgba($grey-6, 0.9);
```
Comment (frankieyan) [inline themis_components/thContextualMessage/index.scss]: Got rid of the transparency - it'll only use `$grey-6` now.

File: themis_components/thAutocomplete/index.scss (hunk 17 above)
Comment (frankieyan) [inline themis_components/thAutocomplete/index.scss]: Removed this whole block for drag-and-drop styles

File: themis_components/thAlert/index.scss [hunk 21]
```diff
@@ -69,7 +69,7 @@ $dismiss-icon-size: 16px;
     }
 
     &.fa-check {
-      background: $green-5;
+      background: $green-dark;
```
Comment (frankieyan) [inline themis_components/thAlert/index.scss]: Called this `$success-color`
